object/invitation.go

```diff
@@ -108,6 +114,10 @@ func UpdateInvitation(id string, invitation *Invitation) (bool, error) {
 }
 
 func AddInvitation(invitation *Invitation) (bool, error) {
+	if _, err := regexp.Compile(invitation.Code); err != nil {
```

Comment:
Define util function: `util.IsValidRegex(s string)`

Reply:
Updated to the following helper functions
```go
func IsRegexp(s string) (bool, error) {
	if _, err := regexp.Compile(s); err != nil {
		return false, err
	}
	return regexp.QuoteMeta(s) != s, nil
}
```